kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/naturesaura/animal_spawner.js

```diff
@@ -182,6 +182,30 @@ onEvent('recipes', (event) => {
             aura: 50000,
             time: 60,
             id: `${id_prefix}duck`
+        },
+        {
+            inputs: [
+                'naturesaura:birth_spirit',
+                Item.of('thermal:satchel').ignoreNBT(),
+                'minecraft:grass',
+                'minecraft:carrot'
+            ],
+            entity: 'alexsmobs:kangaroo',
+            aura: 50000,
+            time: 60,
+            id: `${id_prefix}kangaroo`
+        },
+        {
+            inputs: [
+                'naturesaura:birth_spirit',
+                'upgrade_aquatic:guardian_spine',
```

Comment:
The more I think about it, the guardian spine seems a tiny bit off. Glowing Ink Sacs might be a better option (can obtain glowing squid bucket by pouring starlight on a bucket of regular squid). Prismarine crystals could go well too, given their overal glowy nature. Possibly Nocturnal/Illumination/Stardust powder from Astral.

Not sure here, honestly.

Reply:
Updated the recipes, I like the Nocturnal powder much more, and a live bucket of glow squid instead of a sac, makes it easy to see how to obtain it, and a live one is more 'alluring' to the thrasher lol